com.microsoft.mrtk.graphicstools.unity/Runtime/MeshOutline/MeshSmoother.cs

```diff
@@ -24,16 +24,17 @@ public class MeshSmoother : MonoBehaviour
         private bool smoothNormalsOnAwake = false;
 
         private MeshFilter meshFilter = null;
+        private Mesh originalMesh;
 
         /// <summary>
         /// Helper class to track mesh references.
         /// </summary>
         private class MeshReference
         {
-            public UnityEngine.Mesh Mesh;
+            public Mesh Mesh;

```

Comment:
Could you add the `UnityEngine.` namespace back? We had to add these to fix a namespace conflict with another project that wasn't able to modify their source at the moment. :(

Reply:
Fixed.